### ISSUE TITLE
hellovr_opengl do not work anymore with oculus DK2

### ISSUE THREAD

**Issue by bmarti08** [ValveSoftware/openvr#367]
Hi,

I have a vive and a DK2 to develop an app with openVR. Everything is working fine with the vive but the DK2 seems to not be supported anymore. It is detected by openVR and the position of the HMD is tracked ( the scene is moving on the companion window) but i got nothing more than a grey screen into the HMD.
However exemple from oculus home are working perfectly, so i do not think it come from my drivers or my computer.
I do not remember clearly when the DK2 worked for the last time with the hellovr_openvr sample, but I thinks it was fine two month ago.

Did someone can still play this demo on their DK2 HMD ?

**Comment by Cyphre117** [comment]
I was using a DK2 with the samples around a month ago. if I remember sometimes before displaying anything the DK2 would show a warning saying it's no longer supported which you would have to look at to dismiss. However sometimes this warning would appear behind you.

I know it sounds silly but have you tried turning around while wearing the DK2? It's possible the orientation is messed up and the warning is displaying somewhere offscreen.

**Comment by bmarti08** [comment]
No, I see this message, but it's when I unlock it that the blue/grey screen appear !
I got the same message when I launch oculus home or dreamdesk(a free application on Oculus) but when I unlock here, the house or the game appear well on the DK2.

I only got the problem with openVR and steamVR :/

Anyway, thanks for your reply !

**Comment by fred-dev** [comment]
I have the same problem, the current build does not work with the DK2, it launches the app, but never displays. The DK2 wokrs with the occulus software with the warning and you have ot look at the bar, but after that it is OK. With opnvr I get the same message but never anything on screen, it oges directly to the oculus lobby.
